client/src/components/App.js

```diff
@@ -9,11 +9,12 @@ import SignInForm from "./authentication/SignInForm"
 import TopBar from "./layout/TopBar"
 import GenreIndex from "./genre/GenreIndex.js"
 import AuthenticatedRoute from "./authentication/AuthenticatedRoute"
+import PodcastForm from "./podcasts/PodcastForm.js"
 import UserProfile from "./UserProfile.js"
 import PodcastsIndex from "./podcasts/PodcastsIndex.js"
 import PodcastShowPage from "./podcasts/PodcastShowPage.js"
 import PodcastGenreShow from "./genre/PodcastGenreShow.js"
-import GenreForm from './genre/GenreForm.js' 
+import GenreForm from "./genre/GenreForm.js"
```

Comment:
haha a bit nitpicky, but can you store your Genre componets in a `genres` folder? just to be consistent about pluralization with your podcasts folder